ogg-winmm.c

```
@@ -833,6 +833,58 @@ MCIERROR WINAPI fake_mciSendStringA(LPCSTR cmd, LPSTR ret, UINT cchReturn, HANDL
 			notify = 1; /* storing the notify request */
 			window = (HWND)hwndCallback;
 		}
+		
+		// Handle time format (MM:SS:FF) - check these FIRST before integer formats
+		int from_min, from_sec, from_frame, to_min, to_sec, to_frame;
+		if (sscanf(cmdbuf, "play %*s from %d:%d:%d to %d:%d:%d", &from_min, &from_sec, &from_frame, &to_min, &to_sec, &to_frame) == 6)
+		{
+			
+			DWORD from_ms = from_min * 60000 + from_sec * 1000 + from_frame * 1000 / 75;
```

Don't be angry with me, but I think you should check the `time_format` before converting everything to `ms` then converting them back to `MSF`.

If `time_format` is already `MSF`, you should directly convert them to `MSF`.

If `time_format` is not `MSF`, you should return an error.

---

No trouble, this was anyway copilot-generated late at night without having a clear enough mind to review properly. Just tested to check if it worked properly